Dockerfile

```diff
@@ -94,11 +94,14 @@ CMD /start-cadence.sh
 # All-in-one Cadence server
 FROM cadence-server AS cadence-auto-setup
 
-RUN apk add --update --no-cache ca-certificates py-pip mysql-client
-RUN pip install cqlsh
+RUN apk add --update --no-cache ca-certificates py-pip mysql-client python3
+RUN pip3 install cqlsh
 
 COPY docker/start.sh /start.sh
 
+# test installed applications
+RUN cqlsh --version
```

Comment:
maybe `pip3 install cqlsh && cqlsh --version` so we don't make another layer?
but probably not a bad idea, since the install is far from a guarantee with python.

I suppose it's somewhat odd that there's no support for testing builds in dockerfiles 🤔

Reply:
I think normally you treat them as build artifacts and subject them to e2e testing as you would any other deployable

Reply:
yeah, though in-dockerfile would let you test things as you build rather than only at the end.  e.g. for `from scratch` things, you could test in a build-and-debug-friendly env before you copy the binaries to a minimal image.